nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -76,15 +91,23 @@ void TfOpNode::finalize() {
   };
 
   transform_if(weight_transform, weights);
+  transform_if(input_transform, inputs);
 }
 
 flatbuffers::Offset<void>
 TfOpNode::getBuiltinOps(flatbuffers::FlatBufferBuilder &f) const {
```

Comment:
We can delete the parameter flatbuffers::FlatBufferBuilder &f cause it doesn't need any more.